config/initializers/doorkeeper.rb

```diff
@@ -167,8 +167,12 @@
   # Under some circumstances you might want to have applications auto-approved,
   # so that the user skips the authorization step.
   # For example if dealing with a trusted application.
+  REDIRECT_URI_WHITELIST = [
```

Comment:
Can we have this as an env variable? You should have access to it via: `ENV["REDIRECT_URI_WHITELIST"]`

Reply:
done!